### ISSUE TITLE
New version check popup window has incorrect conda commands

### ISSUE THREAD

**Issue by jeremymatt** [spyder-ide/spyder#15810]
## Problem Description
This is from Spyder version 4.2.5:
The "New Spyder version" window that pops up when there's a new version of Spyder includes the following instruction:
conda update anaconda

This is does not work as I do not have a package called "anaconda" in my current environment.  

Did you mean:
conda update -n base conda

![Screenshot 2021-06-07 19 44 50](https://user-images.githubusercontent.com/32210294/121101811-8f834c80-c7ca-11eb-9580-af53383ba1ea.png)

**Comment by ccordoba12** [comment]
Hey @jeremymatt, thanks for reporting. You said:

> This is does not work as I do not have a package called "anaconda" in my current environment.

Then, please run the second command

    conda install spyder=5.0.0

However, we recommend you not to do that at the moment because that version has a lot of bugs that we've fixed in the last couple of months but sadly Anaconda has not updated to our latest version (5.0.3).

**Comment by jeremymatt** [comment]
This is also an issue in 5.0.0; the "conda update anaconda" instruction is incorrect and should be removed from the popup.  Or if the intent is to have the user upgrade to the latest version of the package manager "conda", the command should be revised to read:
"conda update -n base conda"

I just got the notification that there's a new version available; I have been running into some issues and hopefully the upgrade fixes them.

Thanks for all you do,
-Jeremy

**Comment by dalthviz** [comment]
Note: The command `conda update anaconda` is only valid if you manually installed the `anaconda` package in your env or are using the base env (which usually comes with the anaconda `package`).

Probably that's the thing that should be clarified in the update message.

**Comment by jeremymatt** [comment]
@dalthviz Oh, I hadn't considered that; I install packages into environments as I need them (letting conda handle the dependencies) so I've never really thought about "anaconda" as a set of packages.  I could see updating conda before installing a new spyder version, but does updating anaconda (presumably updating all the packages that come with anaconda?) really get you anything?  Shouldn't the conda solver be able to handle upgrading any packages required by the new version of spyder?

**Comment by dalthviz** [comment]
Hi again @jeremymatt , just in case, the `anaconda` package I was mentioning is this one: https://anaconda.org/anaconda/anaconda and I think it basically works as a placeholder package to install in an env all the package that you see on a base env (so in some sense is a package that works as an snapshot of the versions of the packages that you normally find in a initial base env in an specific moment of time - that's why the versions for the package include a year and a month e.g `2021.11`). My guess for it to exist is to make faster the setup of a base env since there is no need to solve the versions of the different packages that work together but just install them.

So basically updating the `anaconda` package gives you the oportunity to get the most recent snapshot of the packages that you should find in a base env which always is nice because most probably when you will try to update Spyder after, you will have already some dependencies up-to-date installed making the update go faster.

Just in case pinging @ccordoba12 since probably he knows more

**Comment by ccordoba12** [comment]
`conda update anaconda` is only necessary for the base environment. So we could refine the instructions to avoid that in case people has Spyder in other envs or when they installed it with Miniconda.

**Comment by ccordoba12** [comment]
This is quite old and no longer applies.
